### DISCUSSION TITLE
bacterial complexes

### DISCUSSION THREAD

**Discussion by theanita1** [msberends/AMR#82] [General]
Has anyone run an interpretation using complexes rather than specific species?
For example, the AGAR report in Australia it's reported _K. pneumoniae_ complex rather than _K. pneumoniae_, _K. pnuemoniae pneumoniae_, _K. variicola_ as individual bacteria. Same goes for _A. baumanii_ complex and _E. _cloacae_. 

Am currently working around this by making a secondary organism variable where I've renamed the various isolates into their respective complexes, but just thought I'd ask if anyone else is using complexes.

Cheers

**Comment by msberends** [comment]
Great question. Just reading up on AGAR (https://agargroup.org.au), what an AWESOME initiative.

Do you know any literature where the complexes are well-defined? We could add them to the package, but with complexes (and groups) it's hard to find a reliable source of info. For 2.0, we just added all the ~1,500 serotypes of *Salmonella*, but it would be great to also have some complexes (*K. pneumoniae*, *C. freundii*, etc.).
